cypress/page/menu-content.page.ts

```diff
@@ -1,10 +1,12 @@
 class MenuContentPage {
     private tShirtMenu: string;
     private menuContentPageURL: string;
+    private dressesMenu: string;
 
     constructor() {
         this.tShirtMenu = "#block_top_menu > ul > li:nth-child(3) > a";
```

Comment:
In general, it is better not depend on the position of any element (`nth-child`). You can use an attribute of the a tag to improve the selector:
`"#block_top_menu > ul > li > a[title='T-shirts']"`